README.md

```diff
@@ -15,6 +15,15 @@ group :development, :test do
 end
 ```
 
+Want to run against the `master` branch? You'll need to include the dependent
+RSpec repos as well. Add the following to your `Gemfile`:
+
+```ruby
+%w[rspec-rails rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
`rspec-rails` feels out of order here since all the others are in alphabetical order.  Should this go between `rspec-mocks` and `rspec-support`?